### PR TITLE
Web based VSCODE example

### PR DESCRIPTION
**Pull request by MiXaiLL76**
For our work, we use the jupyter + vscode bundle and here is an example of launching it into work.

Of course, it lacks launching "in a new window", as suggested in the PR - https://github.com/trungleduc/jupyter_app_launcher/pull/76

But I think this feature will be implemented soon

### CODE REVIEW

File: samples/code-server/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+**jupyter_lab_config.py**
+
+```py
+from tornado import websocket
+websocket_max_message_size = 1048 * 1024 * 1024
+# fix module
+setattr(websocket, "_default_max_message_size", websocket_max_message_size)
+# fix config
+c.NotebookApp.tornado_settings = {"websocket_max_message_size": websocket_max_message_size}
```
Comment (trungleduc) [inline samples/code-server/README.md:23]:
```suggestion
c.LabApp.tornado_settings = {"websocket_max_message_size": websocket_max_message_size}
```

File: samples/code-server/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+> In the server config, you need to add the following lines
+
+**jupyter_lab_config.py**
+
+```py
+from tornado import websocket
+websocket_max_message_size = 1048 * 1024 * 1024
+# fix module
+setattr(websocket, "_default_max_message_size", websocket_max_message_size)
```
Comment (trungleduc) [inline samples/code-server/README.md:21]: Is WebSocket max size in the tornado setting not enough?

Comment (MiXaiLL76) [reply samples/code-server/README.md:21]: You see, there is 10 MB by default.
Which for example for plotly is extremely small. "doesn't fit".
This caused quite a lot of bugs in my work and server crashes. Both jupyter and vscode. Therefore, expanding this parameter is important.

Comment (trungleduc) [reply samples/code-server/README.md:21]: I understand, but setting it with `c.LabApp.tornado_settings = {"websocket_max_message_size": websocket_max_message_size}` should be enough, I think you don't need to patch the `tornado.websocket` module.

Comment (trungleduc) [reply samples/code-server/README.md:21]: before you use `c.NotebookApp`, which won't work for JupyterLab

Comment (trungleduc) [reply samples/code-server/README.md:21]: ah I see, could you add this link to your comment? It could help other users.

**Review by trungleduc — APPROVED** [review]

**Comment by trungleduc** [comment]
Thanks

**Pull request merged**